scripts/ci/ci-run-stateless-tests-cluster.sh

```diff
@@ -9,4 +9,5 @@ SCRIPT_PATH="$(cd "$(dirname "$0")" >/dev/null 2>&1 && pwd)"
 cd "$SCRIPT_PATH/../../tests" || exit
 
 echo "Starting databend-test"
-./databend-test '^0[^4]_' --mode 'cluster' --run-dir 0_stateless --skip '02_0057_function_nullif' '02_0058_function_ifnull'
+# Now Planner v2 not support cluster execute. So skip some tests that need enable planner v2.
+./databend-test '^0[^4]_' --mode 'cluster' --run-dir 0_stateless --skip '02_0057_function_nullif' '02_0058_function_ifnull' '03_0004_select_order_by_db_table_col_v2'
```

Comment:
Note: the test in this pr need to enable planner v2 but now we do not support cluster execute, so skip it.